### PR TITLE
Sidebar implementation

### PR DESCRIPTION
**Pull request by eggmangrg**
Sidebar functionality implemented into the page 

**Required Items**

- [ ] Title matches [../CONTRIBUTING.md](contributing guidelines)
- [ ] Linked Issue: #XYZ
- [ ] Code Reviewed
- [ ] Manually tested
- Style
  - [ ] No Lint Errors
  - [ ] Quality Comments
- Tests
  - [ ] Unit Tests Added
  - [ ] CI/CD passes
- UI
  - [ ] Picture of before/after UI (if applicable)
image_2022-11-23_195147052

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```
Comment (MacMalainey) [inline src/App.tsx:1]: Why did this file get added back?

**Review by MacMalainey — CHANGES_REQUESTED** [review]
One question, other than that LGTM

File: src/App.tsx (hunk 1 above)
Comment (eggmangrg) [reply src/App.tsx:1]: I synced my fork and then it must have added it back but I am personally not using it.

Comment (MacMalainey) [reply src/App.tsx:1]: Can you please remove that file from your commit then?

Comment (eggmangrg) [reply src/App.tsx]: Removed file

**Review by MacMalainey — APPROVED** [review]

**Pull request merged**